src/main/kotlin/kweb/Kweb.kt

```diff
@@ -271,7 +272,8 @@ class Kweb private constructor(
     }
 
     private suspend fun DefaultWebSocketSession.listenForWebsocketConnection() {
-        val hello = gson.fromJson<Client2ServerMessage>((incoming.receive() as Text).readText())
+        //val hello = gson.fromJson<Client2ServerMessage>((incoming.receive() as Text).readText())
```

Comment:
This is a super-nitpick but we should remove dead code rather than comment.

Reply:
I do this while I am still playing with things. But, in this case, I am done with this dead code, and that line could have been deleted before I committed.